src/views/inspector/InspectorViewProvider.ts

```diff
@@ -366,7 +366,7 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
         const variationName = possibleValue[0]
         return `<div class="detail-entry">
           <label>${variationName}</label>
-          <label class="details-value">${possibleValue[1]}</label>
+          <label class="details-value">${JSON.stringify(possibleValue[1])}</label>
```

Comment:
how is this gonna look for bigger json objects? Might need to truncate at a certain point

Reply:
probably wonky, this will probably happen for long strings as well